### PR TITLE
Fix some errors/typos

### PR DESCRIPTION
**Pull request by sikmir**

### CODE REVIEW

**Comment by nyalldawson** [comment]
All looks good to me - how did you pick these up?

File: src/core/qgseditformconfig.cpp [hunk 1]
```diff
@@ -272,7 +272,7 @@ void QgsEditFormConfig::readXml( const QDomNode& node )
   }
 
   QDomNode editFormInitCodeSourceNode = node.namedItem( QStringLiteral( "editforminitcodesource" ) );
-  if ( !editFormInitCodeSourceNode.isNull() || ( !editFormInitCodeSourceNode.isNull() && !editFormInitCodeSourceNode.toElement().text().isEmpty() ) )
+  if ( !editFormInitCodeSourceNode.isNull() && !editFormInitCodeSourceNode.toElement().text().isEmpty() )
```
Comment (nyalldawson) [inline src/core/qgseditformconfig.cpp]: @m-kuhn this look ok to you?

Comment (m-kuhn) [inline src/core/qgseditformconfig.cpp]: as far as I can see it just removes the redundant `isNull()` check, so yes, I don't see any functional difference but the new version is cleaner.

**Comment by m-kuhn** [comment]
Thank you @sikmir , how did you find these issues?

**Comment by sikmir** [comment]
@nyalldawson, @m-kuhn using static code analyzer ([PVS-Studio](http://www.viva64.com)).

**Pull request merged**